### ISSUE TITLE
Parse Error. Your app's play store page doesn't seem to have latest app version info.

### ISSUE THREAD

**Issue by altick** [kimxogus/react-native-version-check#161]
I've started receiving the following error recently:
`Parse Error. Your app's play store page doesn't seem to have latest app version info.`

I have checked the mobile version of the google play store in a browser using the URL from the source code and can confirm that the current version is present there. However in the returned document reported by the library the current version is missing. The URL seems to be correct, the returned document is indeed the page of the app.

I would appreciate any clues on this. Thank you.

## Technical details
`kimxogus/react-native-version-check` version: 3.4.2
RN: 0.63.4
React: 17.0.2

**Comment by mMarcos208** [comment]
I have the same issue. Any Update?

**Comment by arelstone** [comment]
I was experienced the same issue. I've done a patch see: #162

**Comment by M4Tdev** [comment]
Hi, I have seen that solution but it is based on the backend from another package that is scraping the GP Store on the backend side (https://github.com/flexible-agency/react-native-check-version).

I would assume it still works because the backend gets an old version of Google Play Store and it can scrape the Current Version.

The owner of the package that I mentioned and also the backend API that is suggested in #162 didn't know about the redesign. So I would guess that this method will also break in some time.

Basically, Google made it harder for us to scrape the Store so we have to find a new way to scrape it.

**Comment by ramirezjag00** [comment]
for experimentation purposes, as an alternative to google play store, I tried getting the version in an app's page with `apkcombo.com` and it works

**Comment by Rafael-Pessoni** [comment]
Anyone here knows the solutions for this trouble? I'm getting this error on a published app, and I need to resolve quickly... Please :/

**Comment by jagadyudha** [comment]
Hi, I am also facing this issue

**Comment by FatmaMahmoud698** [comment]
I face the same issue

**Comment by wenge8n** [comment]
Please check [https://github.com/kimxogus/react-native-version-check/issues/93#issuecomment-1160647182](this)

Hope this help

**Comment by FatmaMahmoud698** [comment]
@wenge8n  can you upgrade the packages on the folder react-native-version-check-expo to use this package on expo

**Comment by DavidLozzi** [comment]
Am I correct in understanding that this library scrapes the Google Play page to determine the version available? There isn't an API or some more reliable way to do that on the store? If that's the case, I'll move away from using Google and just store my own version number on my own servers and pull it from my own controlled environment.

**Comment by rf1804** [comment]
Latest version has been released v3.4.3

https://www.npmjs.com/package/react-native-version-check/v/3.4.3

**Comment by jagadyudha** [comment]
> Latest version has been released v3.4.3
> 
> https://www.npmjs.com/package/react-native-version-check/v/3.4.3

Thanks 😀

**Comment by fu-2021** [comment]
react-native-version-check v3.4.4 is the same issue

**Comment by ChauVV** [comment]
> I'm currently having this issue with the latest version: 3.4.7

me too

**Comment by Ali-Aref** [comment]
the same issue on version `3.4.7` any solutions ?

**Comment by BarryFearns** [comment]
Hi All.

This may be obvious but make sure that androidPackageName is correct. I had a typo in mine and got the "Parse Error". After fixing it, it worked straight away so it looks like 3.4.7 is actually working.
